templates/ci/.net-desktop.yml

```diff
@@ -0,0 +1,51 @@
+# .NET Desktop
+# Build and run tests for .NET Desktop or Windows classic desktop solutions.
+# https://aka.ms/yaml
+
+queue: 'Hosted VS2017'
```

Comment:
Considered not quoting strings that don't need to be quoted? This YAML syntax is valid:
```yaml
queue: Hosted VS2017
```

Reply:
We considered that, but decided to quote every string so that people don't have to question whether quotes are needed or not.  Please let me know if it drives you crazy though.